notebooks/2023-02-15-dhs-cross-country-classifier-experiments/2023-02-15_crosscountry_normalized_classifier.ipynb

```diff
@@ -0,0 +1,1507 @@
+{
```

Comment:
<p>@jace would recommend let's not attach interpretations like this. For the dev sector, wealth means a lot of things (income, assets, etc). Even then, our data is asset-based; not income. </p><p> </p><p>Maybe rephrase to something like A = "wealthiest" -&gt; E = "poorest" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/unicef-ai4d-poverty-mapping/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>